sdk/core/src/error/mod.rs

```diff
@@ -39,7 +37,7 @@ impl ErrorKind {
         Self::HttpResponse { status, error_code }
     }
 
-    pub fn http_response_from_body(status: u16, body: &str) -> Self {
+    pub fn http_response_from_body(status: u16, body: &[u8]) -> Self {
```

Comment:
No need to convert a response body to a String. With this change, it should be usable from services.